nym-vpn-core/crates/nym-vpn-lib/src/mobile/runner.rs

```diff
@@ -185,6 +183,9 @@ impl WgTunnelRunner {
             }
         }
 
+        // Shutdown mixnet client.
+        mixnet_client.disconnect().await;
```

Comment:
Note that this is a no-op if it has been built with and external task manager

Reply:
So shutting down the task manager should be enough then right?

Reply:
yes, signal disconnect and possible also wait for it to finish disconnecting